web/src/message_store.ts

```diff
@@ -51,6 +51,7 @@ export type RawMessage = {
     id: number;
     is_me_message: boolean;
     last_edit_timestamp?: number;
+    raw_content?: string;
```

Comment:
Does this belong on `RawMessage`? It’s not sent by the server, but it’s added in `echo.insert_local_message` before it calls `message_events.insert_new_messages` which calls `message_helper.process_new_message`, which I think is where we’re saying `RawMessage` gets converted to `Message`.

Reply:
Yeah, that seems like it should be on `Message`, but probably it can't be cleaned up until we do the `message_helper` conversion, which has been blocked on the actual message type work being messy.

Reply:
Moving it to `Message` doesn’t cause any errors yet. I can do that for now and let this be someone else’s problem.

Reply:
Yeah I am figuring out all the intermediary types that the messages undergo in the app while doing `echo.js` conversion. We most probably should make a type called `LocalEchoedMessage`.

Reply:
Yeah that sounds right to me.